nltk/app/chartparser_app.py

```diff
@@ -746,7 +746,7 @@ def _init_buttons(self, root):
         Frame(buttons, width=20).pack(side="left")
         Button(buttons, text="Swap Charts", command=self._swapcharts).pack(side="left")
 
-        Button(buttons, text="Detatch Output", command=self._detatch_out).pack(
+        Button(buttons, text="Detach Output", command=self._detatch_out).pack(
```

Comment:
Same typo is also in the method name `_detatch_out`

Reply:
Wonderful, thank you! I'm not sure why `codespell` didn't detect it. It did try to update other instances of misspelled functions and methods.